src/API_operations/bookService.js

```diff
@@ -0,0 +1,14 @@
+import http from './http-common';
+
+const getAll = () => http.get('');
+
+const create = (data) => http.post('', data);
+
+const remove = (id) => http.delete(`/${id}`);
```

Comment:
All [Bookstore API](https://www.notion.so/Bookstore-API-51ea269061f849118c65c0a53e88a739) calls are done with `fetch` OR `axios` 💪😎 Love your implementation with http-common file 🤓

Reply:
Thank you very much for your appreciation, please.